tests/integration_tests/performance/test_snapshot_perf.py

```diff
@@ -23,12 +28,12 @@
 # Latencies in milliseconds.
 CREATE_LATENCY_BASELINES = {
     '2vcpu_256mb.json': {
-        'FULL':  1000,
-        'DIFF':  0,
+        'FULL':  270,
```

Comment:
How did we get to these values?

Reply:
Are we sure these values will not trigger intermittent failures ?

Reply:
I updated the test to contain target and delta values. These results were computed based on the empirical rule, which states that 99.7% of data observed following a normal distribution falls within 3 standard deviations of the mean (µ ± 3σ). The delta value was calculated by multiplying the standard deviation by 3.